hubspot3/engagements.py

```diff
@@ -100,3 +100,31 @@ def get_all_as_generator(self, limit=-1, **options):
 
             yield from engagements
 
+    def get_recently_modified(self, start_date: int, end_date: int, **options):
+        """get recently modified engagements"""
```

Comment:
```suggestion
        """Get recently modified engagements."""
```
Acho que tem mais docstrings no código que não seguem o padrão do [PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings), seria bom padronizarmos isso.